suites/quincy/rbd/tier-3_rbd_persistent_write_back_cache.yaml

```diff
@@ -410,3 +411,28 @@ tests:
           desc: RBD Persistent Cache invalidate image level
           module: test_rbd_persistent_writeback_cache_invalidate.py
           name: PWL cache path validation at image level
+
+  - test:
+      abort-on-fail: true
+      config:
+        level: client                        # PWL at client
+        cache_file_size: 1073741824          # 1 GB
+        rbd_persistent_cache_mode: ssd       # "ssd" or "rwl" on pmem device
+        client: node5
+        drive: /dev/nvme0n1
+        cleanup: true
+        validate_exclusive_lock: true
+        rep-pool-only: True
+        rep_pool_config:
+          pool: pool1
+          image: image1
+          size: 10G
+        fio:
```

Comment:
already provided config for `pool name` and `image name` in above rep_pool_config can we use the same.

Reply:
Its just to avoid few lines of code in copying the pool and image to fio parameters. No harm in retaining it this way I feel